### PR TITLE
#37 Keyboard Events (Main Menu)

### PR DESCRIPTION
**Pull request by mathvaillant**

### CODE REVIEW

File: src/hooks/useDeviceBackButtonHandler.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { useEffect } from "react";
+import { BackHandler } from "react-native";
+
+export function useDeviceBackButtonHandler(callback: () => void) {
```
Comment (mathvaillant) [inline src/hooks/useDeviceBackButtonHandler.tsx:4]: [Docs on this BackHandler](https://reactnative.dev/docs/backhandler.

File: src/navigation/authed-stack.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export function AuthedStack() {
       initialRouteName={NavigationScreens.ROUTES}
       screenOptions={{
         header: Header,
-        freezeOnBlur: true,
+        animation: "none",
```
Comment (mathvaillant) [inline src/navigation/authed-stack.tsx:22]: This makes switching between screens almost instantaneous... IMO it is way better like that, as performance is our main goal.

**Comment by mathvaillant** [comment]
[Keyboard Events - Main Menu](https://trello.com/c/Gwq7BES9/37-keyboard-events-main-menu)

**Review by joaoqueiroz95 — APPROVED** [review]

File: src/screens/MainMenu.tsx [hunk 3]
```diff
@@ -4,36 +4,70 @@ import { useNavigation } from "@react-navigation/native";
...
+      "KEYCODE_1",
+      "KEYCODE_AT",
+      "KEYCODE_2",
+      "KEYCODE_A",
+      "KEYCODE_B",
+      "KEYCODE_C",
+      "KEYCODE_0",
+      "KEYCODE_SLASH",
+      "KEYCODE_EQUALS",
+    ],
```
Comment (joel-gouveia) [inline src/screens/MainMenu.tsx:55]: Can we have a file with all the keys to listen for the different screens? that would help us keep track better and reduce noise here

**Review by joel-gouveia — APPROVED** [review]
Besides the small changes requested, looks good

**Pull request merged**